src/main/java/com/amazon/opendistroforelasticsearch/security/support/ConfigConstants.java

```diff
@@ -247,4 +256,12 @@
     public static final List<String> OPENDISTRO_SECURITY_PROTECTED_INDICES_DEFAULT = Collections.emptyList();
     public static final String OPENDISTRO_SECURITY_PROTECTED_INDICES_ROLES_KEY = "opendistro_security.protected_indices.roles";
     public static final List<String> OPENDISTRO_SECURITY_PROTECTED_INDICES_ROLES_DEFAULT = Collections.emptyList();
+
+    public static Set<String> getSettingAsSet(final Settings settings, final String key, final List<String> defaultList, final boolean ignoreCaseForNone) {
```

Comment:
Would be nice if the util function is moved out of ConfigConstants (has only constants). 
This is used only in AuditCategory mainly because of "NONE" dependency which should be removed by #325 . Consider moving it there or address this change with 325 ?

Reply:
It is a shared utility function and I did not find a better place to put it without introducing a new class.

Reply:
Out of curiousity, what are the drawbacks of introducing a new Util class for this function?

Reply:
Every java class requires it's metadata to be loaded into JVM and consumes memory. Not a big deal, but it may be better to avoid introducing a class with a single static method.

Reply:
Thanks for the explanation